### ISSUE TITLE
-v must interpret escape sequences

### ISSUE THREAD

**Issue by ko1nksm** [benhoyt/goawk#129]
Hi, @benhoyt.

`-v` must interpret escape sequences. I hate this spec of awk, but anyway, reality is what it is. ☹️

```console
$ goawk -v VAR='\t' 'BEGIN{printf VAR}' | hexdump -C
00000000  5c 74                                             |\t|

$ mawk -v VAR='\t' 'BEGIN{printf VAR}' | hexdump -C
00000000  09                                                |.|

$ gawk -v VAR='\t' 'BEGIN{printf VAR}' | hexdump -C
00000000  09                                                |.|
```

**Comment by benhoyt** [comment]
Hi @ko1nksm, thanks for your recent bug reports. This is an interesting one. I'll take a look at these in the next few days.

Out of interest, what are you using GoAWK for? Are you using it as a Go library or via the `goawk` command line tool?

**Comment by ko1nksm** [comment]
I am testing if `goawk` (command line tool) can be used as an implementation of awk that [shdotenv](https://github.com/ko1nksm/shdotenv) uses internally.

If you are interested, you can perform the test by following the steps below.

```sh
$ printf "FOO='ABC'" | SHDOTENV_AWK=goawk ./src/shdotenv --no-allexport -e -
FOO='ABC'

# or run all tests
$ shellspec -e AWK=goawk
```

**Comment by benhoyt** [comment]
Thanks for that. I've fixed this via PR #132 (also for the other `var=value` form).
